frontend/src/pages/projects/types.ts

```diff
@@ -62,7 +62,7 @@ export type StorageData = {
 export type StartNotebookData = {
   projectName: string;
   notebookName: string;
-  notebookSize: NotebookSize;
+  notebookSize: NotebookSize | undefined;
```

Comment:
Do not make this change. 
```suggestion
  notebookSize: NotebookSize;
```